src/index.html

```diff
@@ -11,7 +11,207 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+    id="form"
+    name="form"
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    >
```

Comment:
Check the code style everywhere, according to the checklist
```suggestion
    <form
      id="form"
      name="form"
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      method="post"
    >
```